data/human/boarding missions.txt

```diff
@@ -600,3 +600,238 @@ mission "Turn in Tarquin"
+				depart
+			label run
+			`	As you run towards the airlock, explosions erupt on both sides of you, and you're sure that you look like an action hero as you barge through the airlock to safety. Having narrowly escaped death and not wishing to chance it, you quickly depart.`
+				depart
+			label dive
+			`	As you dive towards the airlock, explosions erupt on both sides of you, and you're sure that you look like an action hero as you tuck and roll through the airlock to safety. Having narrowly escaped death and not wishing to chance it, you quickly depart.`
+				depart
+			label success
+			branch beacon
+				random < 15
+			branch virus
+				random < 5
+				"credits" > 1000000
+			action
+				payment 65000
+			`	Despite your best efforts to make the ship actually fly it just won't, but now you have all the time to loot as you please and even pilfer some outfits from the ship. However, before leaving the cockpit you find a credit chip sitting in the cockpit worth <payment>.`
+				decline
+			label beacon
+			action
+				set "derelict pirate beacon"
+			`	However, you soon notice a blinking light on the controls of the ship, it appears to be some kind of beacon. But what could it be signaling? The arrival of pirate ships entering the system gives you a pretty good guess as you race back to your ship.`
+				launch
+			label virus
+			action
+				payment -200000
+			`	However, moments later a screen lights up in the cockpit displaying something strange. Upon closer inspection you realize it's some form of malicious software, and it's attempting to hack into your ship!`
+			`	You race back to your ship, quickly launching and destroying the <origin>, but by the time you manage to destroy the virus it has already hacked into your bank account and stolen <payment> credits from you.`
+				depart
+			label salvage
+			`	Deciding not to risk it you quickly scour the ship for valuable scrap and outfits.`
+			branch valuable
+				random < 40
+			action
+				payment 22500
+			`	Upon a brief inspection of the ship you find little valuable, only managing to salvage a meager <payment> credits from the <origin>. Before you depart you detonate the ship to prevent any other derelict hunters from scouring an empty ship.`
+				depart
+			label valuable
+			branch rare
+				or
+					and
+						random < 25
+						or
+							not "derelict treasure map"
+							not "derelict family investment"
+					random < 5
+			action
+				payment 63000
+			`	Searching through the ship you find that it is quite valuable and you manage to salvage a substantital amount from the <origin>, worth <payment> credits. Before you depart you detonate the ship to prevent any other derelict hunters from scouring the now empty ship.`
```

Comment:
There should probably be a choice of whether to detonate the ship or not.
```suggestion
			`	Searching through the ship you find that it is quite valuable; you manage to salvage a substantial amount from the <origin>, worth <payment> credits. Before you depart you detonate the ship to prevent any other derelict hunters from scouring the now empty ship.`
```